package.json

```diff
@@ -26,6 +26,7 @@
     "gray-matter": "^2.0.1",
     "gulp": "^3.9.0",
     "gulp-concat": "^2.6.0",
+    "gulp-cssnano": "^2.1.0",
```

Comment:
Should we also remove our dependency on the original css-nano or do we still use that?